src/pages/content-edit/versioning.js

```diff
@@ -28,6 +28,15 @@ module.exports = {
 
     view : function(ctrl, options) {
         return m("div", { class : options.class },
+            m("a", {
+                    title  : "Preview",
+                    href   : "/content/" + ctrl.schema + ctrl.id,
```

Comment:
I could be wrong but I _think_ you're incorrect there, since adding a slash just takes you to the page that you're already on?